packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx

```diff
@@ -113,49 +114,50 @@ export const AddObjectMenu = () => {
     [formatMessage],
   );
 
-  const addObjectMenuItems = useMemo(
-    () =>
-      [
-        {
-          icon: { name: 'add-plus' as IconProps.Name },
-          uuid: ObjectTypes.Object,
-        },
-        {
-          uuid: ObjectTypes.Empty,
-        },
-        {
-          uuid: ObjectTypes.Model,
-          isDisabled: !showAssetBrowserCallback,
-        },
-        {
-          uuid: ObjectTypes.EnvironmentModel,
-          isDisabled: !showAssetBrowserCallback || sceneContainsEnvironmentModel,
-          feature: { name: COMPOSER_FEATURES.EnvironmentModel },
-        },
-        {
-          uuid: ObjectTypes.Light,
-        },
-        {
-          uuid: ObjectTypes.ViewCamera,
-          feature: { name: COMPOSER_FEATURES.CameraView },
-        },
-        {
-          uuid: ObjectTypes.Tag,
-        },
-        {
-          uuid: ObjectTypes.Annotation,
-          feature: { name: COMPOSER_FEATURES.Overlay },
-        },
-        {
-          uuid: ObjectTypes.ModelShader,
-          isDisabled: !selectedSceneNodeRef || isEnvironmentNode(selectedSceneNode),
-        },
-        {
-          uuid: ObjectTypes.MotionIndicator,
-        },
-      ].map(mapToMenuItem),
-    [showAssetBrowserCallback, sceneContainsEnvironmentModel, selectedSceneNodeRef, isEnvironmentNode],
-  );
+  const addObjectMenuItems = useMemo(() => {
+    const menuItems = [
+      {
+        icon: { name: 'add-plus' as IconProps.Name },
+        uuid: ObjectTypes.Object,
+      },
+      {
+        uuid: ObjectTypes.Empty,
+      },
+      {
+        uuid: ObjectTypes.Model,
+        isDisabled: !showAssetBrowserCallback,
+      },
+      {
+        uuid: ObjectTypes.EnvironmentModel,
+        isDisabled: !showAssetBrowserCallback || sceneContainsEnvironmentModel,
+        feature: { name: COMPOSER_FEATURES.EnvironmentModel },
+      },
+      {
+        uuid: ObjectTypes.Light,
+      },
+      {
+        uuid: ObjectTypes.Tag,
+      },
+      {
+        uuid: ObjectTypes.Annotation,
+        feature: { name: COMPOSER_FEATURES.Overlay },
+      },
+      {
+        uuid: ObjectTypes.ModelShader,
+        isDisabled: !selectedSceneNodeRef || isEnvironmentNode(selectedSceneNode),
+      },
+      {
+        uuid: ObjectTypes.MotionIndicator,
+      },
+    ];
+    if (!matterportSdk) {
+      menuItems.push({
+        uuid: ObjectTypes.ViewCamera,
+        feature: { name: COMPOSER_FEATURES.CameraView },
+      });
+    }
```

Comment:
you will change the position of this item by pushing as the last element. can we just disable it or need to delete it totally?

Reply:
I wanted to remove this option completely, that's why this code. Is there any issue with change in position?